internal/nostr/nostr.go

```diff
@@ -647,7 +647,7 @@ func (svc *Service) StopSubscriptionHandler(c echo.Context) error {
 
 	svc.Logger.WithFields(logrus.Fields{
 		"subscription_id": subscription.ID,
-	}).Debug("Stopped subscription")
+	}).Info("Stopped subscription")
```

Comment:
we we really need to see this log?

Reply:
We can see this in the db too, but I thought it would be helpful while checking the logs of subscriptions (in future) and we don't log this on stopping a subscription twice, so no overload. (And we will not see these for /nip47)

Reply:
what info does stopped subscription give you when you debug something?

Reply:
It just saves time from having to check in the db while scrolling through logs (using a given `@subscription_id`) to see why it doesn't have more logs after a point (because you see a stopped subscription log), nothing much

Reply:
but isn't that the default? so if nothing is logged it is closed?